addons/website_sale/controllers/cart.py

```diff
@@ -430,8 +439,9 @@ def _get_cart_notification_information(self, order, added_qty_per_line):
                     'image_url': int
```

Comment:
Nit (and unrelated to your change, but would be nice :slightly_smiling_face: ):
```suggestion
                    'image_url': str
```